--- /etc/dovecot/conf.d/10-master.conf  2019-08-26 10:36:36.000000000 +0000
+++ dovecot/conf.d/10-master.conf       2020-12-26 14:24:49.000000000 +0000
@@ -104,9 +104,11 @@
   }

   # Postfix smtp-auth
-  #unix_listener /var/spool/postfix/private/auth {
-  #  mode = 0666
-  #}
+  unix_listener /var/spool/postfix/private/auth {
+    mode = 0666
+    user = postfix
+    group = postfix
+  }

   # Auth process is run as this user.
   #user = $default_internal_user
