--- /etc/dovecot/conf.d/10-ssl.conf     2020-08-05 14:26:17.000000000 +0000
+++ dovecot/conf.d/10-ssl.conf  2020-12-26 14:25:46.000000000 +0000
@@ -9,8 +9,8 @@
 # dropping root privileges, so keep the key file unreadable by anyone but
 # root. Included doc/mkcert.sh can be used to easily generate self-signed
 # certificate, just make sure to update the domains in dovecot-openssl.cnf
-ssl_cert = </etc/dovecot/private/dovecot.pem
-ssl_key = </etc/dovecot/private/dovecot.key
+ssl_cert = </etc/ssl/certs/amazonaws.com.crt
+ssl_key = </etc/ssl/private/amazonaws.com.key

 # If key file is password protected, give the password here. Alternatively
 # give it when starting dovecot with -p parameter. Since this file is often
