--- imapd-ssl_original  2020-12-29 08:58:10.166126473 +0000
+++ imapd-ssl   2020-12-29 09:00:35.126207890 +0000
@@ -227,7 +227,7 @@
 # In all cases, $TLS_CERTFILE needs to be linked to one of the existing
 # certificate files.
 
-TLS_CERTFILE=/etc/courier/imapd.pem
+TLS_CERTFILE=/etc/ssl/certs/amazonaws.com.crt
 
 ##NAME: TLS_PRIVATE_KEYFILE:0
 #
@@ -266,7 +266,7 @@
 # In all cases, $TLS_PRIVATE_KEYFILE needs to be linked to one of the existing
 # certificate files.
 #
-#TLS_PRIVATE_KEYFILE=/etc/courier/imapd_private_key.pem
+TLS_PRIVATE_KEYFILE=/etc/ssl/private/amazonaws.com.key
 
 ##NAME: TLS_DHPARAMS:0
 #
