--- main.cf_original    2020-12-26 21:48:17.439196002 +0000
+++ main.cf     2020-12-26 21:52:19.208574099 +0000
@@ -4,7 +4,7 @@
 # Debian specific:  Specifying a file name will cause the first
 # line of that file to be used as the name.  The Debian default
 # is /etc/mailname.
-#myorigin = /etc/mailname
+myorigin = /etc/mailname
 
 smtpd_banner = $myhostname ESMTP $mail_name (Ubuntu)
 biff = no
@@ -24,8 +24,10 @@
 
 
 # TLS parameters
-smtpd_tls_cert_file=/etc/ssl/certs/ssl-cert-snakeoil.pem
-smtpd_tls_key_file=/etc/ssl/private/ssl-cert-snakeoil.key
+smtpd_tls_cert_file=/etc/ssl/certs/amazonaws.com.crt
+smtpd_tls_key_file=/etc/ssl/private/amazonaws.com.key
 smtpd_tls_security_level=may
 
 smtp_tls_CApath=/etc/ssl/certs
@@ -46,3 +48,4 @@
 inet_interfaces = all
 inet_protocols = ipv4
 html_directory = /usr/share/doc/postfix/html
+home_mailbox = Maildir/
