--- pop3d-ssl_original  2020-12-29 08:58:21.946112447 +0000
+++ pop3d-ssl   2020-12-29 09:01:25.494220858 +0000
@@ -221,7 +221,7 @@
 # In all cases, $TLS_CERTFILE needs to be linked to one of the existing
 # certificate files.
 
-TLS_CERTFILE=/etc/courier/pop3d.pem
+TLS_CERTFILE=/etc/ssl/certs/amazonaws.com.crt
 
 ##NAME: TLS_PRIVATE_KEYFILE:0
 #
@@ -260,7 +260,7 @@
 # In all cases, $TLS_PRIVATE_KEYFILE needs to be linked to one of the existing
 # certificate files.
 #
-#TLS_PRIVATE_KEYFILE=/etc/courier/pop3d_private_key.pem
+TLS_PRIVATE_KEYFILE=/etc/ssl/private/amazonaws.com.key
 
 ##NAME: TLS_DHPARAMS:0
 #
